app/controllers/api_controller/tags.rb

```diff
@@ -34,6 +34,14 @@ def tags_create_resource(parent, _type, _id, data)
       raise BadRequestError, "Could not create a new tag - #{err}"
     end
 
+    def tags_delete_resource(_parent, _type, _id, data)
+      tag_id = parse_id(data, :tags) || parse_by_attr(data, :tags, %w(name))
+      raise BadRequestError, "Tag id, href or name needs to be specified for deleting a tag resource" unless tag_id
+      tag = Tag.find_by_id(tag_id)
```

Comment:
@imtayadeway You'll need to delete the Classification entries here instead of just the tag. There is a before_destroy that calls delete_tags_and_entries. That method deletes the corresponding tags and removes the taggings on any objects that were tagged with the entry.